Use the ha dedicated endpoint (#122)
CrunchyData/bridge-cli: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
### Fixed

> - High availability change with `cb upgrade start`  should be done without any other upgrades.

Reviewer: ```suggestion
- High availability changes with `cb upgrade start` must be made without any other changes to the cluster.
```
